src/partials/header.html

```diff
@@ -1,7 +1,7 @@
 <header class="header">
   <div class="container header__box">
     <a href="#" class="logo">
-      <svg width="45" height="35">
+      <svg class="logo--size" width="45" height="35">
```

Comment:
zmień tą klasę na logo__svg bo korzystamy przecież z BEM'a a to nie jest modyfikator tylko element :P